src/EmbeddedManifold.jl

```diff
@@ -265,13 +265,27 @@ function decorator_transparent_dispatch(
 )
     return Val(:parent)
 end
+function decorator_transparent_dispatch(
+    ::typeof(get_coordinates!),
+    ::AbstractEmbeddedManifold,
+    args...,
+)
+    return Val(:parent)
```

Comment:
Why is this `:parent` instead of `:intransparent`? I think we will usually want people to implement this function for all embedded manifolds.

Reply:
I am nor yet 100% sure, the problem is, that you want to resolve the basis first.

Reply:
What do you mean by resolving the basis first? We already have lots of disambiguation methods to make sure it's correctly resolved.

Reply:
Well, the error was a `VeeOrthogonalBasis` that was not further “passed down” the hierarchy of basis (to orhogonal, then to orhonormal) for the mutating case. This resolving happens usually on the `Manifold` level (i.e. at parent). Of course we can also track down the case where something passes to mutating _before_ resolving the bases, but this way also works.

Reply:
to be precise, it happens when you call `vee!` (or `vee` since that calls `vee!`) so we have to then “look” for the next basis (from `get_coordinates!`) that fits.

Reply:
OK, that seems reasonable.